WindowsServerDocs/administration/server-core/server-core-removed-roles.md

```diff
@@ -23,20 +23,23 @@ The following roles, role services, and features have been removed from the Serv
 - Fax Server (**Fax**)
 - MultiPoint Services (**MultiPointServerRole**)
 - Network Policy and Access Services (**NPAS**)
-- Windows Deployment Services (**WDS**) *(before Windows Server version 1803)*
+- Windows Deployment Services (**WDS**) *
```

Comment:
Looks like the remaining asterisk (`*`) belongs to the italics formatting of the note in parentheses, so it should be removed along with the note.
```suggestion
- Windows Deployment Services (**WDS**)
```

Reply:
Sorry, I just noticed your footnotes, so I will remove the suggestions.